build.sbt

```diff
@@ -25,4 +29,14 @@ publishTo := Some(
     Opts.resolver.sonatypeStaging
 )
 
-ThisBuild / githubWorkflowPublishTargetBranches := Seq.empty
+headerLicenseStyle := LicenseStyle.SpdxSyntax
+headerLicense := {
+  val current = java.time.Year.now().getValue
+  val copyrightYear = startYear.value.fold(current.toString)(start => s"$start-$current")
+  LicenseDetection(
+    licenses.value.toList,
+    organizationName.value,
+    Some(copyrightYear),
+    headerLicenseStyle.value
+  )
+}
```

Comment:
I can be talked out of these to go with the flow.  Without it, the copyright is just 2014, and it drones on for several lines.